client/verta/verta/_cli/deployment/predict.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+import click
+import json
+
+from .deployment import deployment
+from ... import Client
+
+
+@deployment.group()
+def predict():
+    pass
+
+
+@predict.command(name="endpoint")
+@click.argument("path", nargs=1, required=True)
+@click.option("--data", "-d", help="Input for prediction. Must be a valid JSON string.")
+@click.option("--workspace", "-w", help="Workspace to use.")
+def predict_endpoint(path, data, workspace):
+    if not data:
+        raise click.BadParameter("--data must be provided.")  # maybe make this an argument instead?
```

Comment:
I decided to make it a non-optional option (:laughing:) because `-d` matches the syntax of the corresponding cURL, plus all our other `endpoint` CLI commands take the endpoint path as their one-and-only argument, so this felt more consistent.

`click.option` takes a `required` parameter
```suggestion
@click.option("--data", "-d", required=True, help="Input for prediction. Must be a valid JSON string.")
@click.option("--workspace", "-w", help="Workspace to use.")
def predict_endpoint(path, data, workspace):
```

Reply:
Nice. Fixed.